pkgs/desktops/arcan/arcan/default.nix

```diff
@@ -48,13 +48,13 @@
 
 stdenv.mkDerivation rec {
   pname = "arcan" + lib.optionalString useStaticOpenAL "-static-openal";
-  version = "0.6.1";
+  version = "0.6.1.1";
 
   src = fetchFromGitHub {
     owner = "letoram";
     repo = "arcan";
     rev = version;
-    hash = "sha256-2do4+6KB0AAcJk22mN0IA/e/bPaeGipLjI4RSTPqLBg=";
+    sha256 = "1xxx4k7007ljn0r4f6bhhksharc7y8qgxfhdrjq76v46482mmlpr";
```

Comment:
```suggestion
    hash = "sha256-+dJaBSKGbHOwzA26/jDyh2UF9YRwGUcysJIeAM4kvfc=";
```

Reply:
:+1: Is `hash` prefered in general in nixpkgs or is it a preference of yours (I've always used sha256 for some reason).

Reply:
Personal preference. However `hash` is more general and future-proof, because this can be changed in the future if a better hash algorithm is discovered and implemented.

Indeed NetBSD is now using BLAKE2s.